### DISCUSSION TITLE
useTransition list of children components.

### DISCUSSION THREAD

**Discussion by rjhoskins** [pmndrs/react-spring#1817] [Support]
How do I transition the un/&mount in a list situation where I'm fading children components receiving props {text, image, subtext, ....etc}?  Basically, it's an image slider but has text and other _stuff_ going on within the "slide. " 

Not sure what I need to pass to the  useTransition, the array itself or the index or the array[index].  And then inside of my return's  call to transition how I'm passing the current selected index so I get all the juicy details of whatever index I'm at for that array item's data passed into the child component. 

Turns out, just needed to code up the simplest form => [demo ](https://codesandbox.io/s/distracted-haibt-4g09u?file=/src/ChildComponent.jsx)

**Comment by joshuaellis** [comment]
Glad you were able to find your answer.
